bugtool/cmd/configuration.go

```diff
@@ -73,6 +73,11 @@ func defaultCommands(confDir string, cmdDir string, k8sPods []string) []string {
 		"ip rule",
 		"ip -4 route show table 2005",
 		"ip -6 route show table 2005",
+		"ip -4 route show table 200",
+		"ip -6 route show table 200",
+		// xfrm
+		"ip xfrm policy",
+		"ip -s xfrm state",
```

Comment:
@jrfastab @joestringer Too late to the party, but this would dump secret keys, wouldn't it? I'm a bit reluctant to add it, as e.g. secret keys might appear in a customer's sysdump which would allow someone to decrypt customer's traffic.

Reply:
yeah. Lets pull it for now... OTOH I used to debug a CI flake that I couldn't reproduce locally.

Reply:
I'll scrub the lines out so we can still get the statistics.